web-components/src/components/usa-identifier/index.js

```diff
@@ -0,0 +1,156 @@
+import { LitElement, html, css, unsafeCSS } from "lit";
+import uswdsCoreStyle from "@uswds/uswds/scss/uswds-core?inline";
+import usaIdentifierStyle from "@uswds/uswds/scss/usa-identifier?inline";
+
+export class UsaIdentifier extends LitElement {
```

Comment:
Lit recommends using component props as input. [Source → ](https://lit.dev/docs/components/rendering/#:~:text=Use%20only%20the%20component%27s%20properties%20as%20input.)

We should leverage [reactive props](https://lit.dev/docs/components/properties/) if we're able to for defaults / configurable settings.